dom0-updates/qubes-dom0-update

```diff
@@ -336,6 +362,36 @@ else
     fi
 fi
 
+# Switching audio
+
+if [ "$AUDIO_SWITCH" = "pipewire" ]; then
+    if ! rpm -q pipewire pipewire-pulseaudio >/dev/null; then
+        echo "Switching from Pulseaudio to PipeWire" >&2
+        SWITCHING_AUDIO_IN_PROGRESS=yes "$0" "${UPDATEVM_OPTS[@]}" --action=install --allowerasing pipewire pipewire-pulseaudio || exit $?
```

Comment:
I recommend stopping PulseAudio first.

Reply:
This is already done by package scripts.